README.md

```diff
@@ -25,44 +25,105 @@ It correctly bundles React in production mode and optimizes the build for the be
 The build is minified and the filenames include the hashes.<br />
 Your app is ready to be deployed!
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+A netlify [deployment](https://computer-visualizer.netlify.com/) is automatically created for each pull request.
```

Comment:
I'm afraid this is not accurate. The link is I believe only for master. 
Each PR has its own link i.e. https://deploy-preview-9--computer-visualizer.netlify.com/ for PR 9

Reply:
Really? Then what is the point of showing it in a PR?

Reply:
No, we don't need to show it on each PR.
The README won't get touched. So on the frontend page, it would be good if we show the master deployment link.

Reply:
```
A netlify [deployment](https://computer-visualizer.netlify.com/) is automatically created for the latest commit on master.
```